lib/puppet/provider/zabbix.rb

```diff
@@ -49,7 +49,7 @@ def self.create_connection
     zbx = ZabbixApi.connect(
       url: "#{protocol}://#{api_config['default']['zabbix_url']}/api_jsonrpc.php",
       user: api_config['default']['zabbix_user'],
```

Comment:
should we quote all of the attribute here, just to be future safe? Can you please add a comment to that explains why the quotes were added? I'm afraid otherwise they will be removed in the future

Reply:
Also, could we add test(s) for usage of special characters?

Reply:
I'll make the same change for `http_password` - since it uses the same password. I don't think there'd be any benefit for making this change for `user` and `http_user` though?

@kenyon Completely new to specs, and not very familiar with ruby.  I'll take a look at the existing specs and see if I can cobble something together. Would need a good review from someone more familiar with spec writing/ruby though.

Can I test specs locally, without having to push each change here for travis to test?